src/specialOffer/SpecialOffer.tsx

```diff
@@ -0,0 +1,94 @@
+import React from 'react'
+
+import { Button, ButtonStatus } from '../button'
+import { ButtonGroup } from '../buttonGroup'
+import { Disclaimer } from '../disclaimer'
+import { Paragraph } from '../paragraph'
+import { SubHeader } from '../subHeader'
+import { StyledSpecialOffer, StyledSpecialOfferPicture } from './SpecialOffer.style'
+
+type Picture = Readonly<{
+  url: string
+  alt: string
+}>
+
+type Link = Readonly<{
+  label: string
+  url: string
+}>
+
+type Button = Readonly<{
+  label: string
+  onClick: () => void
+}>
+
+type CTALinks = [Link] | [Link, Link]
+type CTAButtons = [Button] | [Button, Button]
+
+type SpecialOfferBaseProps = Readonly<{
+  picture: Picture
+  title: string
+  description: string
+  informationLink?: Link
+}>
+
+export type SpecialOfferLinksProps = Readonly<
+  SpecialOfferBaseProps & {
+    ctaLinks: CTALinks
+  }
+>
+export type SpecialOfferButtonsProps = Readonly<
+  SpecialOfferBaseProps & {
+    ctaButtons: CTAButtons
+  }
+>
```

Comment:
Maybe you have better names in mind instead of `ctaLinks` and `ctaButtons` ?

Reply:
Yeah.. Both are Actions! the only thing that differ is `href` or `onclick` . I would join this API and resolve the element to be used inside the component instead of exposing it to the API

Reply:
Ok I propose  this

```suggestion
export type SpecialOfferLinksProps = Readonly<
  SpecialOfferBaseProps & {
    actions: CTALinks | CTAButtons
  }
>
```

but keeping the `CTALinks` and `CTAButtons` separate types